heat/core/operations.py

```diff
@@ -321,6 +309,13 @@ def sqrt(x, out=None):
     """
     return __local_operation(torch.sqrt, x, out)
 
+def sum(x, axis=None):
+    # TODO: document me
+    # TODO: test me
+    # TODO: make me more numpy API complete
+    # TODO: implement type promotion
```

Comment:
Type promotion has to be generically handled in __reduce_op, TODO can be moved to said function